src/components/AvatarWithImagePicker.js

```diff
@@ -107,6 +110,18 @@ class AvatarWithImagePicker extends React.Component {
         this.setState({isMaxUploadSizeModalOpen: isVisible});
     }
 
+    /**
+     *  Checks if image has valid size and updates avatar
```

Comment:
I'm not sure what "updates avatar" means in this context. Maybe this should just say that it triggers the prop callback?

Reply:
Changed to:  "Checks if the image has the valid size and triggers the onImageSelected callback"